src/Confluent.Kafka/Impl/SafeKafkaHandle.cs

```diff
@@ -1283,10 +1298,15 @@ private void setOption_completionSource(IntPtr optionsPtr, IntPtr completionSour
             setOption_OperationTimeout(optionsPtr, options.OperationTimeout);
             setOption_completionSource(optionsPtr, completionSourcePtr);
 
-            IntPtr[] deleteTopicsPtrs = new IntPtr[deleteTopics.Count()];
+            IntPtr[] deleteTopicsPtrs = new IntPtr[deleteTopics.Count(t => t != null)];
             int idx = 0;
             foreach (var deleteTopic in deleteTopics)
             {
+                if (deleteTopic == null)
```

Comment:
Failing the entire operation with an exception rather than silently ignoring the problem seems better to me, because if the user managed to get a null in here, it's more likely than not they're screwed something up that makes the entire operation flawed. Suggest `ArgumentException` + "Cannot delete topics because one or more topic was specified as null."

Reply:
My original thought was to keep in line with the documentation stating this api is not transactional (some can fail, some can succeed), but your comment about it likely indicating other issues makes sense.  I'll incorporate these fixes tonight.

Reply:
Implemented. Now throws argument exception.